core/src/main/java/xyz/reknown/fastercrystals/commands/impl/FastcrystalsCommand.java

```diff
@@ -37,22 +39,27 @@ public void run(Player player, CommandArguments args) {
                     .orElseGet(() -> pdc.has(key, PersistentDataType.BYTE) && pdc.get(key, PersistentDataType.BYTE) == 0);
         pdc.set(key, PersistentDataType.BYTE, (byte) (toggle ? 1 : 0x0));
 
-        String stateKey = "state." + (toggle ? "on" : "off");
+        String stateKey = "state_" + (toggle ? "on" : "off");
+
+        plugin.saveDefaultConfig();
+        plugin.reloadConfig();
+
+        final String state = Objects.requireNonNullElse(
+            plugin.getConfig().getString(stateKey),
+            stateKey
+        );
+
+        final String text = Objects.requireNonNullElse(
+            plugin.getConfig().getString("text"),
+            "<gold>Turned</gold> <state> <gold>your FastCrystals setting.</gold>"
+        );
```

Comment:
```suggestion
        String text = plugin.getConfig().getString("text");
```
Same as above, can't be null